fire/test_components_py3p5.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
```

Comment:
I've copied this directive from fire/test_components_py3.py, but haven't come across it before and can't find mention of it in pylint's documentation.  If there is a more specific option such as `# Lint as: >python3.5` then that would be better.

Reply:
This is a Google-internal directive.

We'll modify our codebase syncing so its removed from the public codebase, but for now let's keep it so things work internally too.